pkg/eks/api.go

```diff
@@ -274,8 +274,6 @@ func ResolveAMI(provider api.ClusterProvider, version string, ng *api.NodeGroup)
 		resolver = ami.NewAutoResolver(provider.EC2())
 	case api.NodeImageResolverAutoSSM:
 		resolver = ami.NewSSMResolver(provider.SSM())
-	case api.NodeImageResolverStatic:
```

Comment:
Do we maybe want to deprecate the resolver in the sense that we print a message warning users that it will soon stop working? [Like with `utils wait-nodes`](https://github.com/weaveworks/eksctl/commit/8026c48131301c1352a8124124d6af4b16cd3172)?

Reply:
@michaelbeaumont looks like it's already marked as deprecated - https://github.com/weaveworks/eksctl/blob/master/pkg/ctl/cmdutils/nodegroup_flags.go#L41

Are you suggesting an improvement to this?

Reply:
Yes, I think (unless there already is one) printing a warning for a few releases is a good idea.

Reply:
@michaelbeaumont I've added a message as part of the `create nodegroup` command [description](https://github.com/weaveworks/eksctl/blob/fc86d3595803fe3e9434fa0213cd3deec71b87c3/pkg/ctl/create/nodegroup.go#L48). I was not sure if I should mark the flag itself as deprecated.